src/drivers/serac.cpp

```diff
@@ -45,96 +45,117 @@ int main(int argc, char *argv[])
+  args.AddOption(&tz, "-tz", "--traction-z", "Cantilever tip traction in the z direction.");
+  args.AddOption(&slu_solver,
+                 "-slu",
+                 "--superlu",
+                 "-no-slu",
+                 "--no-superlu",
+                 "Use the SuperLU Solver.");
+  args.AddOption(&gmres_solver,
+                 "-gmres",
+                 "--gmres",
+                 "-no-gmres",
+                 "--no-gmres",
                  "Use gmres, otherwise minimum residual is used.");
-  args.AddOption(&lin_params.rel_tol, "-lrel", "--linear-relative-tolerance",
+  args.AddOption(&lin_params.rel_tol,
+                 "-lrel",
+                 "--linear-relative-tolerance",
                  "Relative tolerance for the lienar solve.");
-  args.AddOption(&lin_params.abs_tol, "-labs", "--linear-absolute-tolerance",
+  args.AddOption(&lin_params.abs_tol,
+                 "-labs",
+                 "--linear-absolute-tolerance",
                  "Absolute tolerance for the linear solve.");
-  args.AddOption(&lin_params.max_iter, "-lit", "--linear-iterations",
+  args.AddOption(&lin_params.max_iter,
+                 "-lit",
+                 "--linear-iterations",
                  "Maximum iterations for the linear solve.");
-  args.AddOption(&lin_params.print_level, "-lpl", "--linear-print-level",
-                 "Linear print level.");
-  args.AddOption(&nonlin_params.rel_tol, "-nrel", "--newton-relative-tolerance",
+  args.AddOption(&lin_params.print_level, "-lpl", "--linear-print-level", "Linear print level.");
+  args.AddOption(&nonlin_params.rel_tol,
+                 "-nrel",
+                 "--newton-relative-tolerance",
                  "Relative tolerance for the Newton solve.");
-  args.AddOption(&nonlin_params.abs_tol, "-nabs", "--newton-absolute-tolerance",
+  args.AddOption(&nonlin_params.abs_tol,
+                 "-nabs",
+                 "--newton-absolute-tolerance",
                  "Absolute tolerance for the Newton solve.");
-  args.AddOption(&nonlin_params.max_iter, "-nit", "--newton-iterations",
+  args.AddOption(&nonlin_params.max_iter,
+                 "-nit",
+                 "--newton-iterations",
                  "Maximum iterations for the Newton solve.");
-  args.AddOption(&nonlin_params.print_level, "-npl", "--newton-print-level",
-                 "Newton print level.");
-  args.AddOption(&t_final, "-tf", "--t-final",
-                 "Final time; start time is 0.");
-  args.AddOption(&dt, "-dt", "--time-step",
-                 "Time step.");
+  args.AddOption(&nonlin_params.print_level, "-npl", "--newton-print-level", "Newton print level.");
+  args.AddOption(&t_final, "-tf", "--t-final", "Final time; start time is 0.");
+  args.AddOption(&dt, "-dt", "--time-step", "Time step.");
 
   // Parse the arguments and check if they are good
   args.Parse();
-  if (!args.Good()) {
-    if (myid == 0) {
+  if (!args.Good())
+  {
```

Comment:
Can we go back to the old way on this { setting? I think its the brace wrapping settings.  Everyone's thoughts?

Reply:
Slight preference for the old way, but I'm OK with either as long as it's consistent.

Reply:
I'd prefer to not have separate rules about braces for loops, conditionals, structs, classes, etc. With that in my mind, my vote is braces-on-same-line for everything.